src/main/java/pb360/model/JsonDataModel.java

```diff
@@ -0,0 +1,106 @@
+package pb360.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+@JsonInclude(Include.NON_DEFAULT)
+public class JsonDataModel {
```

Comment:
I explained about JsonDataObject, why it still wrong

Reply:
i will fix it